R/plot_stations.R

```diff
@@ -0,0 +1,67 @@
+#' @title Plot weather stations
+#' @description Plots weather stations on interactive map using `mapview::mapview()` function.
+#'
+#' @details The weather stations are plotted in layers based on the station type. Stations are
+#' also colored by station type. User can click the points to get more info on the stations, for
+#' example the station fmisid. Legends are plotted by default, which can cause problems on small
+#' window. In this case it is recommended to set `legend = FALSE`.
+#'
+#' @param crs Character coordinate projection to use in results.
+#' @param legend A logical for whether to add legends for weather station type to the map. Default
+#' value is `TRUE`.
+#' @importFrom mapview mapview
```

Comment:
These are neat functions but I wonder if they are worth the additional imports. But they might just as well stay in the package if we have no intention to publish this package in CRAN anytime soon.